src/mongocxx/CMakeLists.txt

```diff
@@ -34,39 +34,62 @@ set(LIBMONGOC_REQUIRED_ABI_VERSION 1.0)
 
 set(mongocxx_pkg_dep "")
 
-# Attempt to find libmongoc by new package name (without lib).
-find_package(mongoc-${LIBMONGOC_REQUIRED_ABI_VERSION} ${LIBMONGOC_REQUIRED_VERSION} QUIET)
+if(BUNDLED_MONGOC AND (TARGET mongoc_shared OR TARGET mongoc_static))
+  # If these targets exist, then libmongoc has already been included as a project
+  # sub-directory
+  message ("found libmongoc targets declared in current build scope; version not checked")
+  message ("specify build option -D BUNDLED_MONGOC=OFF to disable")
 
-if(mongoc-${LIBMONGOC_REQUIRED_ABI_VERSION}_FOUND)
-  message ("found libmongoc version ${mongoc-${LIBMONGOC_REQUIRED_ABI_VERSION}_VERSION}")
   if(NOT MONGOCXX_LINK_WITH_STATIC_MONGOC)
-    set(libmongoc_target mongo::mongoc_shared)
+    set(libmongoc_target mongoc_shared)
   else()
-    set(libmongoc_target mongo::mongoc_static)
+    set(libmongoc_target mongoc_static)
   endif()
+  # Ensures availability of mongoc.h via "#include <mongoc.h>"
+  get_property(LIBMONGOC_SRC_DIR TARGET mongoc_shared PROPERTY SOURCE_DIR)
+  get_property(LIBMONGOC_BIN_DIR TARGET mongoc_shared PROPERTY BINARY_DIR)
+  get_property(LIBBSON_SRC_DIR TARGET bson_shared PROPERTY SOURCE_DIR)
+  set(EXTRA_LIBBSON_INCLUDE ${LIBBSON_SRC_DIR}/src/bson)
+  set(EXTRA_LIBMONGOC_INCLUDE ${LIBMONGOC_SRC_DIR}/src/mongoc ${LIBMONGOC_BIN_DIR}/src/mongoc)
 
-  if(MONGOCXX_BUILD_STATIC)
-    set(mongocxx_pkg_dep "find_dependency(mongoc-1.0 REQUIRED)")
+  if(MONGOCXX_BUILD_STATICC)
```

Comment:
```suggestion
  if(MONGOCXX_BUILD_STATIC)
```

Reply:
Ping

Reply:
Strange.  I managed to miss your comment on this the first time.  Fixed now.